testing/coreruleset/coreruleset_test.go

```diff
@@ -250,7 +250,7 @@ SecRule REQUEST_HEADERS:X-CRS-Test "@rx ^.*$" \
 				}
 				fmt.Fprintf(w, urldecodedBody)
 			} else {
-				_, err = io.Copy(w, r.Body)
+				_, err = w.Write(body)
```

Comment:
There shouldn't be any difference in the code but seems previously `writeHeader` would be called twice after this PR. I think it may be an issue in the interceptor but wasn't able to find the issue yet.